app/src/common/urls.js

```diff
@@ -294,6 +294,8 @@ export const URLS = {
     `${urlBase}integration/${projectId}/all/${type}`,
   testIntegrationConnection: (projectId, integrationId) =>
     `${urlBase}integration/${projectId}/${integrationId}/connection/test`,
+  testIntegrationAvailability: (integrationId) =>
```

Comment:
Let's rename to `testIntegrationConntectionGlobal` or `testGlobalIntegrationConntection` as it is used for testing global integration connection.